### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by BenjaminSemah**
## Add Form Validation

In this milestone of the project, we have worked on
- adding form validation to the website.

### CODE REVIEW

File: validation.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  return !email.validity.patternMismatch;
+};
+
+submit.addEventListener('submit', () => {
+  checkLowerCase();
+});
+
+email.addEventListener('input', () => {
+  checkLowerCase();
+});
```
Comment (elyor-doniyorov) [inline validation.js]: While you made a good job implementing the features, however, most of them are not addressed yet. Let's fix them before moving to the next task.

- [ ] The contact form contains at least one validation checking if the email is written in all lowercase:
- If a user submits the contact form with the email field containing uppercase characters an error message should show up.
- If a user submits the contact form with the email field containing only lowercase characters the form is sent normally.

- [ ] The error message should show up close to the "submit" button.
- [ ] The error message fits the design from the chosen template in Figma (e.g., font, colors, images, tests, margins).
- [ ] The form "submit" or the submit button/input "click" event is used to attach the function checking the email format.
- [ ] The students can check if the email is all lowercase by comparing if the email and its lowercase converted version are the same, ie: email === email.toLowerCase(), utilizing regular expressions or looping through each letter of the email string. Any implementation of this check should be considered valid as long as it works properly.

These features of your contact form are not working properly. Kindly, fix it!

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @BenjaminSemah & @matovu-farid, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BenjaminSemah** [comment]
##Add Form Validation
- Correct errors with email validation in forms.

**Review by oloomoses — APPROVED** [review]
Hi team, 

Great job on making all the suggested changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**